.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: python
-python: 2.7
+python:
+  - 2.7
+  - 3.5
```

Comment:
I believe this has the effect of running:
- python 2.7 tox runs everything the tox.ini configures (py27 and py35)
- python 3.5 tox runs everything the tox.ini configures (py27 and py35)

fortunately we can see that in action to confirm